src/projection/projection.tsx

```diff
@@ -2,8 +2,8 @@ import * as React from "react";
 import * as Highcharts from "highcharts";
 import more from "highcharts/highcharts-more";
 import HighchartsReact from "highcharts-react-official";
-import { getRangeData } from "../service";
-import { AreaRangeData } from "./type";
+import { getTimeSeriesData } from "../service";
```

Comment:
TimeSeriesData is too general. I might name it ProjectionData. (Not better to a great extent but at least closer to meaning of this app)

Reply:
I'm regarding the component I created as a Map visualization while yours is a Projection component

Reply:
that makes sense.